samples/dashboard/demos/climate/demo.js

```diff
@@ -76,45 +78,52 @@ async function buildDashboard() {
             chartConstructor: 'mapChart',
             chartOptions: {
                 chart: {
-                    backgroundColor: '#567',
+                    events: {
+                        render: function () {
+
+                            if (!this.styledMode) {
+                                return;
+                            }
+
+                            // force point colors
+                            for (const point of this.series[1].points) {
+                                if (point.graphic && point.color) {
+                                    point.graphic.element.style.fill =
+                                        point.color;
+                                }
+                            }
+
+                            console.log('redraw', this.series[1].points);
```

Comment:
Unnecessary console log- maybe at that point we should enable the `eslint`?

Reply:
Yes, true. I will activate it.